index.html

```diff
@@ -255,17 +306,28 @@ <h2>Urnik za izbrano relacijo</h2>
 	});
 
 	showOfflineWarning = function(){
-		var errorPopup = document.getElementById("offline");
-		errorPopup.style.transition = "opacity 1s ease-in-out";
-		errorPopup.style.opacity = 0;
-		errorPopup.classList.remove("no");
-		setTimeout(function(){errorPopup.style.opacity = 0.95;}, 1000);
+		fadeIn("offline", 1000);
 	}
 
 	hideOfflineWarning = function(){
-		var errorPopup = document.getElementById("offline");
-		setTimeout(function(){errorPopup.style.opacity = 0;}, 1000);
-		setTimeout(function(){errorPopup.classList.add("no");}, 2000);
+		fadeOut("offline", 1000);
+	}
+
+
+	//Fade in and out functions
+
+	function fadeIn(elementID, time){
+		var element = document.getElementById(elementID);
+		element.style.transition = "opacity 1s ease-in-out";
+		element.style.opacity = 0;
+		element.classList.remove("no");
+		setTimeout(function(){element.style.opacity = 0.95;}, time);
```

Comment:
This line here adds a delay between clicking and showing the element. 

Element's opacity is supposed to be 0 before running fadeIn anyway, so just set it to 0.95 directly, without timeouts. The transition effect already has the animation length.